### ISSUE TITLE
Design mockups for File Repository

### ISSUE THREAD

**Issue by kcullion** [kids-first/kf-portal-ui#22]

**Comment by kcullion** [comment]
Specs: https://kidsfirst.atlassian.net/wiki/spaces/KFDRC/pages/215613462/Log+in+and+create+account

**Comment by kcullion** [comment]
Specs
https://kidsfirst.atlassian.net/wiki/spaces/KFDRC/pages/215318550/File+Repository

**Comment by kcullion** [comment]
Presented in meeting today and posted for feedback

**Comment by kcullion** [comment]
![05a kf-file-repository-query-applied](https://user-images.githubusercontent.com/18424138/35628427-e0edaa82-0669-11e8-84dd-f5096d0fe6cc.png)
![08 kf-file-repository-advanced-selected](https://user-images.githubusercontent.com/18424138/35628428-e0fa86f8-0669-11e8-9972-ae0999ac86cf.jpg)

**Comment by kcullion** [comment]
A couple mockups are attached for the file repository style

**Comment by kcullion** [comment]
![05a kf-file-repository-query-applied](https://user-images.githubusercontent.com/18424138/35635634-d375d998-067c-11e8-817a-3531d351ebb4.png)
![08 kf-file-repository-advanced-selected](https://user-images.githubusercontent.com/18424138/35635635-d381c636-067c-11e8-9046-98477a3bb04b.jpg)

A couple mockups are attached for the file repository interface and overlay style

**Comment by kcullion** [comment]
Implementing changes from team:
- Keep only button “Download manifest” without the option to bundle
- The “lighter” version of File Repo: show top 10, stats at the top of the table, adding some padding between the 3 panels

**Comment by kcullion** [comment]
![kidsfirst-file-repository-v3-artboardsfile repository - queried](https://user-images.githubusercontent.com/18424138/35756535-ba63f58e-0839-11e8-8fdc-7efbd19a436a.jpg)
![kidsfirst-file-repository-v3-artboardsadvanced search overlay](https://user-images.githubusercontent.com/18424138/35756536-bd0f0c60-0839-11e8-8d5d-be1721635baa.jpg)



We made some changes to the File Repository mockups today, in order to simplify the interface and align it more with the MVP features. Attached is a pdf of the changes:
1. Added Github icon in the footer
2. Moved the table counts above the table instead of in the right panel
3. Modified the File actions to reflect MVP
4. Included a single file download option in the table
